pkg/componentreadiness/jiraautomator/jiraautomator.go

```diff
@@ -0,0 +1,597 @@
+		description := `Component Readiness has found a potential regression in the following tests:`
+		for i, test := range tests {
+			// Only show stats for the worst regression
+			if i == 0 {
+				description += fmt.Sprintf("\n h4. Most Regressed Test:\n{code}%s{code}\n", test.TestName)
+				description += getProbabilityString(test.ReportStatus, *test.FisherExact)
+				description += getStatsString("Sample (being evaluated)", test.SampleStats, view.SampleRelease.RelativeStart, view.SampleRelease.RelativeEnd)
+				description += getStatsString("Base (historical)", *test.BaseStats, view.BaseRelease.RelativeStart, view.BaseRelease.RelativeEnd)
+				if len(tests) > 1 {
+					description += fmt.Sprintf("\n h4. Other Regressed Tests:\n")
+					description += fmt.Sprintf("\nThe following tests are also regressed in the same component readiness report. They might not be related to the most regressed test above. We only create one issue per component and therefore group them here. Feel free to create new issues if they are unrelated.\n")
+				}
+			} else {
+				description += fmt.Sprintf("{code}%s{code}\n", test.TestName)
+				description += getProbabilityString(test.ReportStatus, *test.FisherExact)
+			}
+		}
+		if linkedIssue != nil {
+			description += fmt.Sprintf("\n h4. Potentially Related Issues:\n")
+			description += fmt.Sprintf("\n* This regression might be related to [%s|%s]. Feel free to link it if found related.\n", linkedIssue.Key, linkedIssue.Self)
+		}
+
+		absUrl, viewUrl, err := j.getComponentReadinessURLsForView(view)
+		if err != nil {
+			return nil, err
+		}
+		description += fmt.Sprintf("\n h4. Useful Component Readiness Links:\n")
+		description += fmt.Sprintf("\nWe are proving the following two links for your convenience:\n")
+		description += fmt.Sprintf("\n- Click [here|%s] to access the component readiness page generated at the time this issue was created.\n", absUrl)
+		description += fmt.Sprintf("\n- Click [here|%s] to access the component readiness page that will be generated at the time when it is clicked. This is useful for developers to verify their fixes.\n", viewUrl)
+
+		summary := fmt.Sprintf("Component Readiness: %s test regressed", component)
+		issue := jira.Issue{
+			Fields: &jira.IssueFields{
+				Description: description,
+				Type: jira.IssueType{
+					Name: "Bug",
+				},
+				Project: jira.Project{
+					Key: jiratype.ProjectKeyOCPBugs,
+				},
+				Components: []*jira.Component{
+					{
+						Name: component,
+					},
+				},
+				Summary: summary,
+				AffectsVersions: []*jira.AffectsVersion{
+					{
+						Name: view.SampleRelease.Release,
+					},
+				},
+				Labels: []string{jiratype.LabelJiraAutomator},
+			},
+		}
+		created, err := j.jiraClient.CreateIssue(&issue)
+		if err != nil {
+			return created, err
+		}
+		// Set Release Blocker field. Jira does not allow setting those during creation. So do it in separate step.
+		return j.updateReleaseBlocker(created, view.SampleRelease.Release)
```

Comment:
I think we need to add an Instructions section. Things like how we expect teams to handle breaking out the jiras, do not close this jira or another one will be automatically created, if/when we'll automatically close this jira, any other guidelines we want to convey.

Reply:
How about the following:

Recommendations

This is an automatically generated Jira card against the component based on stats generated from component readiness dashboard. Here are some useful recommendations when dealing with this card:
1. It is best to use this card as a placeholder for all the regressed tests for your component. A linked issue can be created for each regressed test. Please only close this issue when all known regressed tests are believed fixed.
2. Please do not remove “Release Blocker” label. The bot will automatically add it back if regressed tests continues showing for the component.

Anything else?